Testing/SystemTests/lib/systemtests/stresstesting.py

```diff
@@ -563,23 +577,28 @@ def start(self, script):
 #########################################################################
 class TestScript(object):
 
-    def __init__(self, test_dir, module_name, test_cls_name):
+    def __init__(self, test_dir, module_name, test_cls_name, exinpr):
```

Comment:
This is a horrible variable name. How about `exclude_in_pr_builds` or `excludeInPullRequests`?